frontend/src/screens/learner.tsx

```diff
@@ -21,23 +21,51 @@ const HighlightedStudies: FC = () => {
     const { highlightedStudies } = useParticipantStudies()
     const isMobile = useIsMobileDevice()
 
+    const handleClick = () => {
+        const element = document.getElementById('all-studies-unique-id')
+        element?.scrollIntoView({ behavior: 'smooth' })
+    }
+
     if (!highlightedStudies.length) return null
 
     return (
         <Box bg={colors.navy} py='md'>
             <Container>
-                <Stack>
-                    <Title c='white' order={2}>Highlighted Studies</Title>
-                    {isMobile ?
-                        <MobileStudyCards studies={highlightedStudies} /> :
-                        <DesktopStudyCards studies={highlightedStudies} />
-                    }
-                </Stack>
+                <Flex direction='column'>
+                    <Flex justify='space-between'>
+                        <CuratedStudies />
+                        {isMobile ?
+                            <MobileStudyCards studies={highlightedStudies} /> :
+                            <DesktopStudyCards studies={highlightedStudies} />
+                        }
+                    </Flex>
+                    <Flex c={colors.green} justify='center' align='center'>
+                        <Flex direction='column' justify='center' align='center' style={{ cursor: 'pointer' }} onClick={()=> handleClick()}>
+                            <Text size='sm'>View all studies</Text>
+                            <IconChevronDown size='.875rem'/>
+                        </Flex>
+                    </Flex>
+                </Flex>
             </Container>
         </Box>
     )
 }
 
+const CuratedStudies: FC = () => {
+
+    return (
+        <Flex c={colors.white} direction='column' w='280px' h='350px' pt='2.25rem' mt='1rem'>
```

Comment:
Just make sure this container looks okay on mobile

Additionally, remember to try and use <Stack> or <Group> where possible as they are a bit easier to see at a glance what's going on

Reply:
Okay